### PR TITLE
[TRUS-2064] - Un-map Trustee Individual

### PR DESCRIPTION
**Pull request by jassalrichy**
Combine Trustee Individual and Trustee Organisations into Trustees Extractor

### CODE REVIEW

File: app/models/playback/http/GetTrust.scala [hunk 1]
```diff
@@ -154,9 +154,9 @@ case class DisplayTrustLeadTrusteeOrgType(
                                            name: String,
                                            phoneNumber: String,
                                            email: Option[String] = None,
-                                           identification: DisplayTrustIdentificationOrgType,
+                                           identification: Option[DisplayTrustIdentificationOrgType],
```
Comment (adamconder) [inline app/models/playback/http/GetTrust.scala]: Identification is a required field in the response, what's the reason this was made optional?

File: app/mapping/playback/TrusteesExtractor.scala [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        answers.set(TrusteeUtrYesNoPage(index), false)
+          .flatMap(answers => extractAddress(address.convert, index, answers))
+
+      case DisplayTrustIdentificationOrgType(_, Some(utr), None) =>
+        answers.set(TrusteeUtrYesNoPage(index), true)
+          .flatMap(_.set(TrusteesUtrPage(index), utr))
+
+    } getOrElse {
+      Logger.error(s"[TrusteesExtractor] no identification for lead trustee company returned in DisplayTrustOrEstate api")
+      case object InvalidExtractorState extends RuntimeException
```
Comment (adamconder) [inline app/mapping/playback/TrusteesExtractor.scala]: This case object is defined elsewhere, can we import and reuse that one?

File: conf/messages.en [hunk 3]
```diff
@@ -1016,7 +1044,9 @@ answerPage.section.trustsDetails.heading = Trust details
 answerPage.section.deceasedSettlor.heading = Settlor
 answerPage.section.settlors.heading = Settlors
 answerPage.section.settlor.subheading = Settlor {0}
+answerPage.section.leadTrustee.subheading = Lead trustee
 answerPage.section.trustees.heading = Trustees
+answerPage.section.leadTrusteeIndividual.heading = Lead Trustee
```
Comment (adamconder) [inline conf/messages.en]: Do we need both Lead Trustee header definitions?

File: app/models/playback/http/GetTrust.scala (hunk 1 above)
Comment (jassalrichy) [reply app/models/playback/http/GetTrust.scala]: Reverted back to non optional.

File: app/mapping/playback/TrusteesExtractor.scala (hunk 2 above)
Comment (jassalrichy) [reply app/mapping/playback/TrusteesExtractor.scala]: imported

File: conf/messages.en (hunk 3 above)
Comment (jassalrichy) [reply conf/messages.en]: removed redundant message file entry "answerPage.section.leadTrusteeIndividual.heading"

**Pull request merged**